# 请求字段
string name
float32 x
float32 y
float32 z
float32 theta_x
float32 theta_y
float32 theta_z
---
# 响应字段
bool success
string message